Argo/Extensions/RawRepresentable.swift

```diff
@@ -0,0 +1,30 @@
+public extension Decodable where Self.DecodedType == Self, Self: RawRepresentable, Self.RawValue == String {
+  // Work around beta 6 bug where types cannot be deduced properly:
+  private typealias Stringish = Self.DecodedType
+  public static func decode(json: JSON) -> Decoded<Self> {
```

Comment:
can we change `json` to `j`? that follows the style for the rest of the standard types.